sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -525,46 +505,44 @@ public void run() {
      * @param batchId      The batch ID.
      */
     @MainThread
-    private synchronized void sendLogs(final GroupState groupState, final int currentState, List<Log> batch, final String batchId) {
-        if (checkStateDidNotChange(groupState, currentState)) {
+    private void sendLogs(final GroupState groupState, final int currentState, List<Log> batch, final String batchId) {
```

Comment:
is currentState parameter unused?, should cleanup if yes.

Reply:
No, it's used for calling `checkPendingLogsAfterPost`